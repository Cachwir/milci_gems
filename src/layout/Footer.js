import React from "react";

function Footer()
{
    return (
        <div>
            {/* Footer / Start */}
            <footer id="footer">
                {/* 960 Container */}
                <div className="container">

                    {/* About */}
                    <div className="four columns">
                        <img id="logo-footer" src={process.env.PUBLIC_URL + "/assets/theme/images/logo-footer.png"} alt="" />
                        <p>Propulsé par ReactJS.</p>
                        <p>Images et textes par Milci.</p>
                    </div>

                    {/* Contact Details */}
                    <div className="four columns">
                        <h4>Coordonnées du développeur</h4>
                        <ul className="contact-details-alt">
                            <li><i className="halflings white user"></i> <p><strong>Discord:</strong> Cachwir#3822</p></li>
                            <li><i className="halflings white envelope"></i>
                                <p><strong>Email:</strong> cachwir@gmx.com</p></li>
                        </ul>
                    </div>

                </div>
                {/* 960 Container / End */}

            </footer>
            {/* Footer / End */}


            {/* Footer Bottom / Start  */}
            <footer id="footer-bottom">

                {/* 960 Container */}
                <div className="container">

                    {/* Copyrights */}
                    <div className="eight columns">
                        <div className="copyright">
                            © Copyright 2020 par Cachwir. Tous droits réservés.
                        </div>
                    </div>

                </div>
                {/* 960 Container / End */}

            </footer>
            {/* Footer Bottom / End */}
        </div>
    );
}

export default Footer;