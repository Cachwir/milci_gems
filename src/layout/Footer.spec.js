import { shallow } from 'enzyme'
import { expect } from 'chai'
import React from 'react'
import Footer from './Footer'

describe('<Footer />', () => {
    it('should match its reference snapshot', () => {
        const wrapper = shallow(
            <Footer />
        );

        expect(wrapper).to.matchSnapshot();
    });
});