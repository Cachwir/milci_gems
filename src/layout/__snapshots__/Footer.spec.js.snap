// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match its reference snapshot 1`] = `
<div>
  <footer
    id="footer"
  >
    <div
      className="container"
    >
      <div
        className="four columns"
      >
        <img
          alt=""
          id="logo-footer"
          src="/assets/theme/images/logo-footer.png"
        />
        <p>
          Propulsé par ReactJS.
        </p>
        <p>
          Images et textes par Milci.
        </p>
      </div>
      <div
        className="four columns"
      >
        <h4>
          Coordonnées du développeur
        </h4>
        <ul
          className="contact-details-alt"
        >
          <li>
            <i
              className="halflings white user"
            />
             
            <p>
              <strong>
                Discord:
              </strong>
               Cachwir#3822
            </p>
          </li>
          <li>
            <i
              className="halflings white envelope"
            />
            <p>
              <strong>
                Email:
              </strong>
               cachwir@gmx.com
            </p>
          </li>
        </ul>
      </div>
    </div>
  </footer>
  <footer
    id="footer-bottom"
  >
    <div
      className="container"
    >
      <div
        className="eight columns"
      >
        <div
          className="copyright"
        >
          © Copyright 2020 par Cachwir. Tous droits réservés.
        </div>
      </div>
    </div>
  </footer>
</div>
`;
